stable/aerospike/values.yaml

```diff
@@ -0,0 +1,96 @@
+# Default values for aerospike.
+terminationGracePeriodSeconds: 30
+replicaCount: 1
+nodeSelector: {}
+image:
+  repository: aerospike/aerospike-server
+  tag: 3.14.1.2
+  pullPolicy: Always
+
+# Set as empty object {} if no volumes need to be created
+# See confFile below
+persistentVolume: {}
+  # - mountPath: /opt/aerospike/data
+  #   name: aerospike-data
+  #   template:
+  #     accessModes: [ "ReadWriteOnce" ]
+  #     # storageClassName:  "standard"
+  #     resources:
+  #       requests:
+  #         storage: "36G"
+  #     selector:
+  #       matchLabels:
+  #         diskname: "aerospike-data"
+
+service:
+  type: ClusterIP
+  annotations: {}
+  loadBalancerIP:
+  clusterIP: None
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #  cpu: 100m
+  #  memory: 128Mi
+  # requests:
+  #  cpu: 100m
+  #  memory: 128Mi
+
+confFile: |-
+  #default config file
+  service {
+      user root
+      group root
+      paxos-single-replica-limit 1
+      pidfile /var/run/aerospike/asd.pid
+      service-threads 4
+      transaction-queues 4
+      transaction-threads-per-queue 4
+      proto-fd-max 15000
+  }
+  logging {
+      file /var/log/aerospike/aerospike.log {
```

Comment:
In principle with Kubernetes we don't need to log to files anymore. I think it should be enough to log to the console.